jobs/furnishings/src/furnishings/config.py

```diff
@@ -145,6 +145,7 @@ class TestConfig(_Config):  # pylint: disable=too-few-public-methods
 
     # BCLaws SFTP
     BCLAWS_SFTP_STORAGE_DIRECTORY = 'bclaws'
+    BCLAWS_SFTP_PRIVATE_KEY = ''
```

Comment:
In `post_processor` the `base64.b64decode` expects a string value. If it is not set by the `.env` it will receive a NoneType, throwing an error. I set an empty string as for the `TestConfig` to prevent this.